src/components/playerstats/playerstats.js

```diff
@@ -173,6 +173,10 @@ import ServerConnections from '../ServerConnections';
                     value: session.TranscodingInfo.TranscodeReasons.map(translateReason).join('<br/>')
                 });
             }
+            sessionStats.push({
```

Comment:
I think it would be a good idea to wrap this in an if like the stats above, just in case.

Reply:
bit of a nitpick but visually I think it will nicer if it's above the transcoding reasons.

Reply:
I'm not sure if that would work out since this variable will always be set to something and always display. Not too sure if this is desired behavior though, will wait on more feedback!

Reply:
Fair point! It should be rare enough with mismatching web and server versions for this to be an issue.

Reply:
Everything else is wrapped in `if`. We should follow the "standard".

Reply:
Sorry for the delay, just accepted the change!